### ISSUE TITLE
I can't set seed as null

### ISSUE THREAD

**Issue by dossorio** [hautelook/AliceBundle#209]
When I set `seed` as `null` as instructed in the docs (`seed: 1             # A seed to make sure faker generates data consistently across runs, set to null to disable`):
```yaml
hautelook_alice:
    db_drivers:
        mongodb: true
    locale: "%locale%"
    seed: null
```
I get:
```
[Symfony\Component\Config\Definition\Exception\InvalidTypeException]
  Invalid type for path "hautelook_alice.seed". Expected int, but got NULL.
  Hint: A seed to make sure faker generates data consistently across runs, set to null to disable
```

**Comment by andrewjt71** [comment]
Hi, I've addressed this issue https://github.com/hautelook/AliceBundle/pull/255. Hope this helps

**Comment by theofidry** [comment]
Closed via #255
